File: Selenium/Activity5.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.WebElement;

public class Activity5 {

	public static void main(String[] args) {
		
WebDriver driver=new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-controls");
		
		String title=driver.getTitle();
		System.out.println("Title of the page is: "+title);
		
		WebElement checkbox=driver.findElement(By.id("checkbox"));
		
		driver.findElement(By.xpath("//button[text()='Toggle Checkbox']")).click();
		System.out.println("Is Checkbox displayed: "+checkbox.isDisplayed());
		
		driver.findElement(By.xpath("//button[text()='Toggle Checkbox']")).click();
		System.out.println("Is Checkbox displayed: "+checkbox.isDisplayed());
		
		driver.quit();
		
		

	}

}

File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    title=driver.title
    print("Title of the page is: ",title)

    checkbox=driver.find_element(By.ID,"checkbox")

    driver.find_element(By.XPATH, "//button[text()='Toggle Checkbox']").click()
    print("Is Checkbox displayed: ", checkbox.is_displayed())

    driver.find_element(By.XPATH, "//button[text()='Toggle Checkbox']").click()
    print("Is Checkbox displayed: ", checkbox.is_displayed())

